### PR TITLE
client:Implemented the enqueuing mechanism for clients

### PR DESCRIPTION
**Pull request by ritwik-69**
# fixes: #60 

## Description
Created two functions that pushes the event object into the event queue of a specific client id.
 The second Function sends the each event of the given client id to all the eligible client id (i.e only if response object is available).

## Motivation and Context
[Explain the motivation behind these changes and provide any relevant context.]

## How to Test
[Describe the steps to test the changes made in this pull request.]

## Related Issues
[If applicable, mention any related issues or pull requests.]

## Checklist
- [ ] I have tested these changes locally.
- [x] I have reviewed the code and ensured it follows the project's coding guidelines.
- [ ] I have updated the documentation, if necessary.
- [ ] I have assigned reviewers to this pull request.

## Screenshots (if applicable)
[If your changes include any visual updates, provide screenshots here.]

### CODE REVIEW

File: backend/eventRecipients.ts [hunk 1]
```diff
@@ -31,9 +31,19 @@ export function getClient(clientId: ClientId) {
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function scheduleSend(clientId: ClientId, event: AppEvent) {
     //todo: Enqueue the event for sending.
+    eventQueue.get(clientId)?.push(event);
 }
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function doSendEvent(clientId: ClientId) {
```
Comment (kuv2707) [inline backend/eventRecipients.ts:38]: This is not a part of this PR.

File: backend/eventRecipients.ts [hunk 2]
```diff
@@ -31,9 +31,19 @@ export function getClient(clientId: ClientId) {
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 export function scheduleSend(clientId: ClientId, event: AppEvent) {
     //todo: Enqueue the event for sending.
+    eventQueue.get(clientId)?.push(event);
```
Comment (kuv2707) [inline backend/eventRecipients.ts:34]: You can remove the todo comment and the eslint directive

**Comment by ritwik-69** [comment]
@kuv2707 done with the changes

**Comment by ritwik-69** [comment]
@kuv2707 edited the commit message.

**Comment by kuv2707** [comment]
@ritwik-69 This can be merged once the merge conflicts are resolved.

**Comment by kuv2707** [comment]
Merged, thanks @ritwik-69

**Pull request merged**